distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -552,7 +552,8 @@ private static Integer findPortNumber(Integer portToTry) {
                 + "http://svn.apache.org/repos/asf/servicemix/m2-repo@id=servicemix,"
                 + "http://repository.springsource.com/maven/bundles/release@id=springsource,"
                 + "http://repository.springsource.com/maven/bundles/external@id=springsourceext,"
-                + "http://oss.sonatype.org/content/repositories/releases/@id=sonatype"),
+                + "http://oss.sonatype.org/content/repositories/releases/@id=sonatype,"
+                + "http://artifacts.codice.org/content/repositories/releases"),
```

Comment:
I think we might need to include http://artifacts.codice.org/content/repositories/thirdparty in order to get the desired artifacts to resolve properly